pkg/kubectl/cmd/edit.go

```diff
@@ -430,9 +468,9 @@ func RunEdit(f *cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args
 				}
 				return nil
 			}
-
-			// loop again and edit the remaining items
-			infos = results.edit
```

Comment:
`infos` will be overwritten immediately at the start of next iteration in the for loop. See: [this line](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/edit.go#L200)
So this line is actually useless.